x-pack/plugins/uptime/public/components/queries/empty_state/empty_state.tsx

```diff
@@ -41,22 +48,38 @@ export const EmptyState = ({
             <EuiEmptyPrompt
               title={
                 <EuiTitle size="l">
-                  <h3>No Uptime Data</h3>
+                  <h3>
+                    <FormattedMessage
+                      id="xpack.uptime.emptyState.title"
+                      defaultMessage="No Uptime Data"
+                    />
+                  </h3>
                 </EuiTitle>
               }
               body={
                 <Fragment>
-                  <p>There is no uptime data available.</p>
+                  <p>
+                    <FormattedMessage
+                      id="xpack.uptime.emptyState.bodyMessage"
+                      defaultMessage="There is no uptime data available."
+                    />
+                  </p>
                   <p>
                     {
                       <EuiLink
                         target="_blank"
                         href="https://www.elastic.co/guide/en/beats/heartbeat/6.5/configuring-howto-heartbeat.html"
                       >
-                        Configure Heartbeat
+                        <FormattedMessage
+                          id="xpack.uptime.emptyState.bodyLink"
+                          defaultMessage="Configure Heartbeat"
+                        />
                       </EuiLink>
                     }
-                    &nbsp;to start logging uptime data.
+                    <FormattedMessage
+                      id="xpack.uptime.emptyState.bodyLinkPredicateText"
+                      defaultMessage=" to start logging uptime data"
```

Comment:
The order of words can be defferent in other languages. That's why we should try to provide the whole sentence. In this case we can place the link in `values` like in example below

```
<FormattedMessage
   id="xpack.uptime.emptyState.configureHeartbeatToStartMessage"
   defaultMessage="{configureHeartbeatLink} to start logging uptime data"
   values={{
       configureHeartbeatLink: (
              <EuiLink
                   target="_blank"
                   href="https://www.elastic.co/guide/en/beats/heartbeat/6.5/configuring-howto-heartbeat.html"
               >
                    <FormattedMessage
                          id="xpack.uptime.emptyState.configureHeartbeatToStartMessage.configureHeartbeatLinkText"
                          defaultMessage="Configure Heartbeat"
                    />
             </EuiLink>
        )
   }}
/>
```

Reply:
Addressed in https://github.com/elastic/kibana/pull/28183/commits/b3b5fbc87454585e2553db7a35f1f354421d4dd4.